apps/experiments/models.py

```diff
@@ -691,6 +691,42 @@ def is_public(self) -> bool:
     def is_participant_allowed(self, identifier: str):
         return identifier in self.participant_allowlist or self.team.members.filter(email=identifier).exists()
 
+    def version_details_for_display(self) -> list[dict]:
```

Comment:
nit: this is a good place to use a TypedDict or a dataclass

Reply:
Good idea: https://github.com/dimagi/open-chat-studio/pull/675/commits/a585408e1d250ad84136c5dc35e88629a76c6166